.bash_nav

```diff
@@ -7,3 +7,31 @@ alias desk='cd /mnt/c/Users/gsera/Desktop'
 # source bash
 
 alias srcbash='source ~/.bashrc'
+
+# ---This alias effectively creates a bookmarking app---
+# it adds global symlink shortcuts for use with the 'cds' command
+# ex: 'cds studycloud' will run 'cd -P ~/.short/studycloud'
+# to add a shortcut: go to the target directory and run 'cds + <shortcut_name>'
+# to remove a shortcut: run 'cds - <shortcut_name>'
+cds() {
+    if [ ! -d ~/.short ]; then
+        mkdir ~/.short
+    fi
+    if [ $# -eq 0 ]; then
+        ls ~/.short/
+    else
+        if [ $1 = "+" ]; then
+            ln -s "$PWD" "$HOME/.short/$2"
+        elif [ $1 = "-" ]; then
+            rm ~/.short/$2
+        else
+            if [ -d ./$1 ]; then
+                cd $1
+            elif [ -L ~/.short/$1 ]; then
+                cd -P ~/.short/$1
```

Comment:
Jump to the directory that is pointed to by the symlink by calling cd -P on it.
This will have the effect of cd-ing to the actual, physical location of the symlink's target.